src/components/ModelingSidebar/ModelingSidebar.tsx

```diff
@@ -292,7 +298,7 @@ function ModelingPaneButton({
       <button
         className="group pointer-events-auto flex items-center justify-center border-transparent dark:border-transparent disabled:!border-transparent p-0 m-0 rounded-sm !outline-0 focus-visible:border-primary"
         onClick={onClick}
-        name={paneConfig.title}
+        name={typeof paneConfig.title === 'string' ? paneConfig.title : ''}
```

Comment:
Please get the inner text content of the component here, instead of omitting it. That name is used for accessibility and the tooltip I believe.

Reply:
Oh nice idea, yeah absolutely

Reply:
Turns out there's no easy way to do this. But also in those cases we just make sure title is a string when it's needed anyway. We can revisit this but it's not worth doing hacks just to get some text right now.